pkg/known/label.go

```diff
@@ -15,3 +15,9 @@ const (
 	AnalyticsUidLabel  = "analysis.crane.io/analytics-uid"
 	AnalyticsTypeLabel = "analysis.crane.io/analytics-type"
 )
+
+const (
```

Comment:
where we need those labels?

Reply:
To know the recommendation from a RecommendationRule by label selector. for example ` kubectl crane` use it to get recommendation result.

Reply:
got it.